source/includes/aws/_objects.md

```diff
@@ -2,7 +2,100 @@
+|-----------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `id`<br/>*string*                 | The ID of the object, which is of the form ":regionName/:bucketName/:objectName".                                                                                                                                                |
+| `name`<br/>*string*               | The name of the object.                                                                                                                                                     
+| `fullPath`<br/>*string*               | The actual key of the object inside the bucket, which is represented as a full path.                                                                                                                                                    |
+| `size`<br/>*integer*     | The size of the object in bytes.                                                                                                                                                     |
+| `storageClass`<br/>*string*            | The Type of storage class on the object.                                                                                                                                                      |
+| `lastModified`<br/>*string*       | The date the last time the object was updated.
+| `displaySize`<br/>*string*       | The size of the object represented as an object in either KB/MB/GB.
+| `contentType`<br/>*string*       | The type of file.
+| `eTag`<br/>*string*       | The eTag of the object generated by S3 during initial upload.
+| `isFolder`<br/>*boolean*       | Boolean denoting whether the object actr as a folder. This is always true when the name ends with a '/' character.
+| `bucketName`<br/>*string*       | The name of the bucket the object exists within.
+| `region`<br/>*string*       | The name of the region the object exists within.
+| `isVirtual`<br/>*string*       | Boolean denoting whether the folder actually exists (there exists a key in the bucket named 'folder/'), or just because it's part of the path of a complete object path ('folder/object'). In this request, if nested=true, it will not return any virtual folders due to the nature of obtaining common prefixes and nested objects.
+
+| Mandatory Query Paramaters                        | &nbsp;                                                                                                                                                                                                                   |
```

Comment:
```suggestion
| Required Query Paramaters                        | &nbsp;                                                                                                                                                                                                                   |
```
to keep it consistent within the docs

Reply:
https://github.com/cloudops/cloudmc-api-docs/commit/be42623c5dd36eda0a92e0993af4077e0ada4b37